quickwit-cli/Cargo.toml

```diff
@@ -38,6 +38,7 @@ tokio = { version = "1", features = ["full"] }
 tokio-util = { version = "0.6", features = ["full"] }
 crossterm = "0.20"
 atty = "0.2"
+tantivy = { git= "https://github.com/quickwit-inc/tantivy", rev="ff631ec", default-features=false, features = ["mmap", "lz4-compression"] }
```

Comment:
You had to add tantivy as a deps of quickwit-cli because of the construction of the FileSlice.
Let's either add a From<> implem in tantivy or add an extra constructor to BundleStorage.

Reply:
I tried the From<> version but there is a conflict with the generic From impl
```
impl<B> From<B> for FileSlice
where
    B: StableDeref + Deref<Target = [u8]> + 'static + Send + Sync,
```